tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -78,34 +83,41 @@ var actionCreateCmd = &cobra.Command{
     _, _, err = client.Actions.Insert(action, sharedSet, false)
     if err != nil {
       whisk.Debug(whisk.DbgError, "client.Actions.Insert(%#v, %t, false) error: %s\n", action, sharedSet, err)
-      errMsg := fmt.Sprintf("Unable to create action: %s", err)
+      errMsg := fmt.Sprintf(
+        wski18n.T("Unable to create action: {{.err}}",
+          map[string]interface{}{"err": err}))
       whiskErr := whisk.MakeWskErrorFromWskError(errors.New(errMsg), err, whisk.EXITCODE_ERR_NETWORK,
         whisk.DISPLAY_MSG, whisk.NO_DISPLAY_USAGE)
       return whiskErr
     }
 
-    fmt.Fprintf(color.Output, "%s created action %s\n", color.GreenString("ok:"), boldString(action.Name))
+    fmt.Fprintf(color.Output,
+      wski18n.T("{{.ok}} created action {{.name}}\n",
+        map[string]interface{}{"ok": color.GreenString("ok:"), "name": boldString(action.Name)}))
```

Comment:
Correct.  I believe "ok" is one of those unique abbreviations that is universal and requires no translation.  At least that's how it was handled in prior projects I've worked on.